conftest.py

```diff
@@ -25,7 +25,8 @@
 logger = logging.getLogger(__name__)
 
 
-def pytest_sessionstart():
+@pytest.fixture(scope="session", autouse=True)
+def testdata():
     """ Download sct_testing_data prior to test collection. """
```

Comment:
I guess this docstring is no longer true, then. :sweat_smile:

Reply:
Aha! I've figured out a hook we can use that solves the problems of both `pytest_sessionstart` and `pytest.fixture`: [`pytest_collection`](https://docs.pytest.org/en/stable/reference.html#pytest.hookspec.pytest_collection)

> You can implement this hook to only perform some action before collection, for example the terminal plugin uses it to start displaying the collection counter (and returns None).

This will get called only once (regardless of number of parallel processes), but before tests are collected.